src/main.cc

```diff
@@ -798,6 +798,20 @@ int main(int argc, char* argv[])
                    { "clear", { false, "clear dead sessions" } },
                    { "ro", { false, "readonly mode" } } }
     };
+
+    if (argc == 2 and String(argv[1]) == "--help")
```

Comment:
Actually I didn't only do this PR to have a nice summary of the CLI parameters available, but to also follow the long help option convention (most if not all utilities implement this, regardless of the amount of minus signs they use for other options).